docs/content/docs/tutorials/memorydb-example.md

```diff
@@ -0,0 +1,288 @@
+lead: "Create and use MemoryDB for Redis using Amazon Elastic Kubernetes Service (EKS)."
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 42
+toc: true
+---
+
+The ACK service controller for Amazon MemoryDB for Redis lets you manage MemoryDB Cluster directly from Kubernetes.
+
+This guide will show you how to create and connect to [MemoryDB for Redis](https://aws.amazon.com/memorydb/) through Kubernetes.
+
+## Setup
+
+Although it is not necessary to use Amazon Elastic Kubernetes Service (Amazon EKS) with ACK, this guide assumes that you have access to an Amazon EKS cluster. If this is your first time creating an Amazon EKS cluster, see [Amazon EKS Setup](https://docs.aws.amazon.com/deep-learning-containers/latest/devguide/deep-learning-containers-eks-setup.html). For automated cluster creation using `eksctl`, see [Getting started with Amazon EKS - `eksctl`](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html) and create your cluster with Amazon EC2 Linux managed nodes.
+
+### Prerequisites
+
+This guide assumes that you have:
+
+- Created an EKS cluster with Kubernetes version 1.18 or higher.
+- Setup the [Amazon VPC Container Network Interface (CNI) plugin for Kubernetes](https://docs.aws.amazon.com/eks/latest/userguide/managing-vpc-cni.html) for the EKS Cluster.
+- AWS IAM permissions to create roles and attach policies to roles.
+- Installed the following tools on the client machine used to access your Kubernetes cluster:
+  - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv1.html) - A command line tool for interacting with AWS services.
+  - [kubectl](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html) - A command line tool for working with Kubernetes clusters.
+  - [eksctl](https://docs.aws.amazon.com/eks/latest/userguide/eksctl.html) - A command line tool for working with EKS clusters.
+  - [Helm 3.7+](https://helm.sh/docs/intro/install/) - A tool for installing and managing Kubernetes applications.
+
+### Install the ACK service controller for MemoryDB
+
+You can deploy the ACK service controller for Amazon MemoryDB using the [memorydb-chart Helm chart](https://gallery.ecr.aws/aws-controllers-k8s/memorydb-chart). You can download it to your workspace using the following command:
+
+```bash
+helm pull oci://public.ecr.aws/aws-controllers-k8s/memorydb-chart --version=v0.0.1
+````
+
+You will need to decompress and extract the Helm chart. You can do so with the following command:
+
+```bash
+tar xzvf memorydb-chart-v0.0.1.tgz
+```
+
+You can now use the Helm chart to deploy the ACK service controller for Amazon MemoryDB to your EKS cluster. At a minimum, you need to specify the AWS Region to execute the MemoryDB API calls.
+
+For example, to specify that the MemoryDB API calls go to the `us-east-1` region, you can deploy the service controller with the following command:
+
+```bash
+helm install memorydb-chart --generate-name --set=aws.region=us-east-1
+```
```

Comment:
Please update the code an documentation to reflect the latest Helm 3.8 CLI options:

```
# Log into the chart repository
aws ecr-public get-login-password --region us-east-1 | helm registry login -u AWS --password-stdin public.ecr.aws

# Install the memorydb-chart Helm chart
helm install \
    --generate-name \
    --namespace ack-system \
    --set aws.region=us-east-1 \
    oci://public.ecr.aws/aws-controllers-k8s/memorydb-chart \
    --version v0.0.1
```

Reply:
I have kept the Helm3.7 specific instructions for now.
I think it can be updated later across all the examples files.